region/wreckedship/main/Wrecked Ship East Super Room.json

```diff
@@ -276,6 +309,26 @@
         "If Phantoon is alive, 'Base' can be used instead.",
         "Note that this strat is useless in vanilla, as the item only spawns if Phantoon has been killed."
       ]
+    },
+    {
+      "link": [2, 1],
+      "name": "WS East Supers R-Mode Standup Clip (Right to Left)",
+      "notable": true,
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        "canCeilingClip",
+        "canBePatient",
+        "Morph"
+      ],
+      "note":  [
+        "In R-Mode, kill the Coverns until there is Energy in Samus's Reserves. Get into the Morph tunnel and go to the far left.",
```

Comment:
S'pose the Player would refill on the coverns before and after the clip.  So canBePatient fits.